src/Commands/MigrateFileToDatabaseCommand.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Nwidart\Modules\Commands;
+
+use Illuminate\Console\Command;
+use Symfony\Component\Console\Input\InputOption;
+
+class MigrateFileToDatabaseCommand extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'module:migrate-to-database';
```

Comment:
Maybe change the `module:migrate-to-database`

Reply:
Could you suggest a name?

Reply:
Currently, I don't have a better name, however, it feels weird to type, could just be me.